bin/c8.js

```diff
@@ -26,7 +26,10 @@ let argv = buildYargs().parse(instrumenterArgs)
       await promisify(fs.mkdir)(argv.tempDirectory, { recursive: true })
     }
 
-    process.env.NODE_V8_COVERAGE = argv.tempDirectory
+    if (!process.env.NODE_V8_COVERAGE) {
```

Comment:
Which should c8 take precedence, when both `NODE_V8_COVERAGE` environment variable and `--tmp-directory`? The current implementation will choose `NODE_V8_COVERAGE` in this case.

Reply:
`--tmp-directory` should win, but we default it to `NODE_V8_COVERAGE` if not set, we can get rid of this `if` statement probably

Reply:
> we can get rid of this `if` statement probably

👍